static/src/javascripts/projects/common/views/acquisitions-epic-buttons.html

```diff
@@ -0,0 +1,17 @@
+<div class="contributions__amount-field">
+  <div>
+    <a class="contributions__option-button contributions__contribute contributions__contribute--epic contributions__contribute--epic-member contributions__contribute--epic-member-top"
+      href="<%=membershipUrl%>"
```

Comment:
that goes on to be passed to the buttons template [here](https://github.com/guardian/frontend/pull/17376/files#diff-c95b6c3bf1bf127107da04f082848002R62) - as an object, rather than separate params